notification/store.go

```diff
@@ -169,6 +170,13 @@ func NewStore(ctx context.Context, db *sql.DB) (*Store, error) {
 			from outgoing_messages om
 			where message_type = $1 and contact_method_id = $2 and created_at >= $3
 		`),
+		svcInfo: p.P(`
+			SELECT
+				name,
+				(SELECT count(*) FROM alerts WHERE service_id = $1 AND status = 'triggered')
+			FROM services
+			WHERE id = $1
+		`),
```

Comment:
```suggestion
```

missed this last time but this was only used by the function you just removed; i don't think this is needed either

Reply:
thanks I missed this too, this isn't needed anymore